Pages/homePage.js

```diff
@@ -19,49 +31,66 @@ class HomePage{
         return ".category-item"
     }
 
-    get fashionCategory(){
-        return browser.driver.findElement(by.css("div.category-item a"));
-    }
-
     get logoutButton(){
         return browser.driver.findElement(by.css("div.login-acccount button"));
     }
+
+    get createAnAccount(){
+        return browser.driver.findElement(by.css("a.header-text"));
+    }
+    get firstItemForLinks(){
+        return browser.driver.findElement(by.css(".arrivals div div div div a"));
+    }
+
+    get featureCollectionItems(){
+        return browser.driver.findElement(by.css(".feature-collection div div div div div a")); 
+    }
     
     // ACTIONS
+    
+    clickonFeatureCollectionItem(){
+        console.log("This method clicks on first item of Feature Collection");
+        return this.featureCollectionItems.click();
+    }
 
-    waitForCategories(){
-        console.log("This method waits for Category to show")
-        browser.wait(EC.presenceOf($(this.categoryLocator)), 5000)
+    clickonFirstItemForLinks(){
+        console.log("This method clicks on first item of New Arrival, Top Rated, Last Chance links");
+        return this.firstItemForLinks.click();
     }
 
-    waitForLoginButton(){
```

Comment:
Should waitForLoginButton be inside HomePage ?
Does it belong to Page ?

Reply:
Sorry, you're right, it belongs to Page.js, and I have already moved getter method, I forgot to move the method itself